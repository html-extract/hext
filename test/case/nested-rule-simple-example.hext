<div class:class id>
  {
    <p @text:content />
  }
</div>
