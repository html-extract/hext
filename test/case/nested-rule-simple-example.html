<html>
  <body>
    <div id="list1" class="list1">
      <div class="item"><p>One</p>  </div>
      <div class="item"><p>Two</p>  </div>
      <div class="item"><p>Three</p></div>
    </div>
    <div id="list2" class="list2">
      <div class="item"><p>Four</p> </div>
      <div class="item"><p>Five</p> </div>
      <div class="item"><p>Six</p>  </div>
    </div>
  </body>
</html>
