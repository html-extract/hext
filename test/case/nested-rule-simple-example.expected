{"class":"list1","content":["One","Two","Three"]}
{"class":"list2","content":["Four","Five","Six"]}
